data/create-tables.js

```diff
@@ -0,0 +1,39 @@
+const client = require('../lib/client');
+const { getEmoji } = require('../lib/emoji.js');
+
+// async/await needs to run in a function
+run();
+
+async function run() {
+
+  try {
+    // initiate connecting to db
+    await client.connect();
+
+    // run a query to create tables
+    await client.query(`
+                CREATE TABLE users (
+                    id SERIAL PRIMARY KEY,
+                    email VARCHAR(256) NOT NULL,
+                    hash VARCHAR(512) NOT NULL
+                );           
+                CREATE TABLE todo (
```

Comment:
convention would be 'todos' just like above 'users' totally fine as is but you'll want a plural in the future.